source/_components/light.tikteck.markdown

```diff
@@ -8,8 +8,8 @@ comments: false
 sharing: true
 footer: true
 ha_category: Light
-ha_iot_class: "Local Polling"
-ha_logo: tikteck.png
```

Comment:
Is this correct?

Reply:
`ha_logo` is invalid, `ha_iot_class` is a duplicate key.